### PR TITLE
Perform a round of numerous bugfixes

### PR DESCRIPTION
**Pull request by s-heppner**
This fixes bugs and findings after a preliminary review of the v3.0 implementations. 

Note, that before v3.0 can be considered done, the following issues remain:

- The AAS/Aas, XSD/Xsd abbreviation mess (#177)
- Data_specification_physical_unit changes, that are still yet being specified at this moment (#188)

### CODE REVIEW

File: aas_core_meta/v3.py [hunk 1]
```diff
@@ -5544,22 +5611,27 @@ class Data_specification_IEC_61360(Data_specification_content):
...
+        
     """
 
     value_list: Optional["Value_list"]
     """
     List of allowed values
     """
 
-    value: Optional[str]
+    value: Optional["Lang_string_short_name_type_iec_61360"]
```
Comment (BirgitBoss) [inline aas_core_meta/v3.py]: Why is the name of the type "Lang_string_short_name_type_iec_61360" and not just "Short_name_type_iec_61360" as specified? Similar for other types starting with Lang_string

File: aas_core_meta/v3.py [hunk 2]
```diff
@@ -5280,28 +5280,28 @@ class Level_type(DBC):
...
+    max: "bool"
+    """Maximum of the value"""
 
     def __init__(
         self,
         min: "bool",
-        max: "bool",
         nom: "bool",
         typ: "bool",
+        max: "bool",
```
Comment (BirgitBoss) [inline aas_core_meta/v3.py:5355]: why did you change the order? the order is min, max, nom, typ

**Review by BirgitBoss — CHANGES_REQUESTED** [review]
see questions

**Comment by mristin** [comment]
Closing since it will be fixed/merged in a series of separate pull requests.

**Pull request closed without merging**